users/migrations/0001_initial.py

```diff
@@ -1,9 +1,8 @@
-# Generated by Django 2.2.13 on 2020-09-22 08:03
+# Generated by Django 2.2.21 on 2021-05-07 05:36
```

Comment:
Editing old migrations is prohibited if there isn't a really really good reason to do so. This affects developers' environments and the server environments as there have been already done migrations and changing will take extra time to do changes these manually.